go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/360EntSecGroup-Skylar/excelize/v2
+module github.com/artiz/excelize
```

Comment:
the incorrect package name in the go module.

Reply:
yep, package with default name in go.mod https://github.com/artiz/excelize/tree/extended-parse-time could not be installed by `go mod` so it was temporary fix, will revert it